web/src/main/java/no/nav/modiapersonoversikt/legacy/sak/service/saf/SafGraphqlServiceImpl.kt

```diff
@@ -198,10 +215,19 @@ class SafGraphqlServiceImpl : SafService {
         isLogiskDokument = false
         variantformat = getVariantformat(dokumentInfo)
         skjerming = getSkjerming(dokumentInfo)?.toString()
-
+        dokumentStatus = getDokumentStatus(dokumentInfo)
         return this
     }
 
+    private fun getDokumentStatus(dokumentInfo: HentBrukersDokumenter.DokumentInfo): Dokument.DokumentStatus =
+        when (dokumentInfo.dokumentstatus) {
+            HentBrukersDokumenter.Dokumentstatus.UNDER_REDIGERING -> Dokument.DokumentStatus.UNDER_REDIGERING
+            HentBrukersDokumenter.Dokumentstatus.FERDIGSTILT -> Dokument.DokumentStatus.FERDIGSTILT
+            HentBrukersDokumenter.Dokumentstatus.KASSERT -> Dokument.DokumentStatus.KASSERT
+            HentBrukersDokumenter.Dokumentstatus.AVBRUTT -> Dokument.DokumentStatus.AVBRUTT
+            else -> throw RuntimeException("Ugyldig tekst for mapping til dokumentstatus. Tekst: ${dokumentInfo.dokumentstatus}")
```

Comment:
Else-blokka her er vel for å håndtere om `dokumentstatus == null` siden alle enum-verdiene er etter det jeg kan se tatt hensyn til.
Isåfall kan vi i henhold til dokumentasjonen av feltet sette dokumentstatus til `FERDIGSTILT`:
> # Dokumentstatus gir en indikasjon på hvorvidt dokumentet er ferdigstilt eller under arbeid, eventuelt avbrutt. Dersom dokumentet ikke har noen dokumentstatus, er dokumentet komplett / ferdigstilt.